templates/plugin/dynflow_core.yml.erb

```diff
@@ -0,0 +1,61 @@
+---
+# Path to dynflow database, leave blank for in-memory non-persistent database
+:database: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::database_path') %>
+:console_auth: <%= scope.lookupvar('::foreman_proxy::plugin::dynflow::console_auth') %>
+
+# URL of the foreman, used for reporting back
+:foreman_url: <%= scope.lookupvar("foreman_proxy::foreman_base_url") %>
+
+# SSL settings for client authentication against foreman.
+<% if [nil, :undefined, :undef].include?(scope.lookupvar("foreman_proxy::foreman_ssl_ca")) -%>
+#:foreman_ssl_ca: ssl/certs/ca.pem
+<% else -%>
+:foreman_ssl_ca: <%= scope.lookupvar("foreman_proxy::foreman_ssl_ca") %>
+<% end -%>
+<% if [nil, :undefined, :undef].include?(scope.lookupvar("foreman_proxy::foreman_ssl_cert")) -%>
+#:foreman_ssl_certificate: ssl/certs/fqdn.pem
+<% else -%>
+:foreman_ssl_cert: <%= scope.lookupvar("foreman_proxy::foreman_ssl_cert") %>
+<% end -%>
+<% if [nil, :undefined, :undef].include?(scope.lookupvar("foreman_proxy::foreman_ssl_key")) -%>
+#:foreman_ssl_private_key: ssl/private_keys/fqdn.pem
+<% else -%>
+:foreman_ssl_key: <%= scope.lookupvar("foreman_proxy::foreman_ssl_key") %>
+<% end -%>
+
+# Listen on address
+:listen: 127.0.0.1
```

Comment:
More testing showed listening on 127.0.0.1 is not enough, as when needing to use `fqdn` as the callback_url, it might not resolve to `127.0.0.1` and the connection doesn't happen. I suggest defaulting to `0.0.0.0` and relying on the certs/firewall to prevent unwanted access to the service.